src/main/java/seedu/address/model/Model.java

```diff
@@ -71,6 +71,16 @@ public interface Model {
      */
     void deletePerson(Person target);
 
+    /**
+     * Adds a new task to taskList
+     */
+    void addTask(Task task);
+
```

Comment:
`addTask()` and `hasTask()` are refactored into `setTask()` does not affect your code.

Reply:
#89